src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,44 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp,liked, onLikeToggle}) => {
+
+  const handleHeartClick = () => {
+    onLikeToggle(id);
+  };
+
+  let isLocalUser = false
+
+  if (sender === 'Vladimir') {
+    isLocalUser = true;
+  } 
```

Comment:
```suggestion
  } ;
```

Linter calls out that you're missing semi-colons. if you haven't configured your linter, you might want to do that so it can highlight syntax/style issues.